webhook/browsertrix.go

```diff
@@ -0,0 +1,290 @@
+package webhook
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	urlpkg "net/url"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/lestrrat-go/jwx/v2/jwt"
+	"github.com/starlinglab/integrity-v2/aa"
+	"github.com/starlinglab/integrity-v2/config"
+	wacz "github.com/starlinglab/integrity-v2/preprocessor/wacz"
+	"github.com/starlinglab/integrity-v2/util"
+)
+
+type BrowsertrixCrawlFinishedResponse struct {
+	OrgID     string `json:"orgId"`
+	ItemID    string `json:"itemId"`
+	Resources []struct {
+		Name        string `json:"name"`
+		Path        string `json:"path"`
+		Hash        string `json:"hash"`
+		Crc32       int    `json:"crc32"`
+		Size        int    `json:"size"`
```

Comment:
```suggestion
		Size        int64    `json:"size"`
```

For correctness we should explicitly support file sizes above 4 GiB.

Reply:
updated and also updated comparison